asciidoc-confluence-publisher-cli/src/it/java/org/sahli/asciidoc/confluence/publisher/cli/ArgumentsParserTest.java

```diff
@@ -0,0 +1,77 @@
+package org.sahli.asciidoc.confluence.publisher.cli;
+
+import org.junit.Test;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.*;
+
+public class ArgumentsParserTest {
+
+    ArgumentsParser parser = new ArgumentsParser();
+
+    @Test
+    public void optionalArgument_returnsEmptyIfValueNotPresent() {
```

Comment:
Code-Style: we usually follow the triple-A testing style with `methodUnderTest_relevantState_expectation` (i.e. `optionalArgument_valueNotPresent_returnsEmptyOptional` so the relevant state is clearly visible (also, we usually use `// arrange` , `// act` and `// assert` comments to separate the aspects, i.e. in `AsciidocConfluencePageTest`)

Reply:
great naming strategy. fixed